data-es7.js

```diff
@@ -159,7 +159,14 @@ exports.tests = [
   link: 'http://wiki.ecmascript.org/doku.php?id=harmony:observe',
   category: 'draft',
   exec: function () {/*
-    return typeof Object.observe === 'function';
+    var O = {x: 1};
```

Comment:
Let's change `O` to something like `obj` coz it's almost confusing with `0`. Other than this, LGTM, and thanks, as always!